src/app/containers/Video/index.jsx

```diff
@@ -51,6 +55,41 @@ const VideoContainer = ({ blocks }) => {
     },
   ];
 
+  const env = process.env.APP_ENV
+    ? process.env.APP_ENV
+    : process.env.STORYBOOK_APP_ENV;
+  const mediatorURL = {
+    test: 'open.test.bbc.co.uk',
+    live: 'open.bbc.co.uk',
+    local: 'open.test.bbc.co.uk',
+  };
+
+  // prettier-ignore
+  const mediaPlayerSettings = {
+    product: 'news',
+    responsive: true,
+    statsObject: { clipPID: pid },
+    playlistObject: {
+      title,
+      holdingImageURL: `https://${holdingImageUrl}`,
```

Comment:
The holding image URL actually shouldn't have the protocol for it to work with SMP. I'm not sure why, but it doesn't work if you pass in the protocol
```suggestion
      holdingImageURL: holdingImageUrl,
```

Reply:
@sareh does this work for you in storybook? When I remove the protocol none of them work 💭

Reply:
It did, and then now it doesn't. Interesting. Feel free to ignore this point.